src/core/css/inc/mixins.scss

```diff
@@ -219,17 +219,8 @@ $default-arrow-margin: 3px;
     font-weight: 400;
   }
 
-  // This is required because a <button> element with a :hover effect
-  // that changes its appearance will transition to the hover state onClick
-  // when using a mobile browser.
-  // This is the cause of
-  // https://github.com/mozilla/addons-frontend/issues/1904
-  //
-  // For more info, see: https://css-tricks.com/annoying-mobile-double-tap-link-issue/#article-header-id-2
-  @media (pointer: fine) {
```

Comment:
I kept testing it and it didn't really apply globally. It was very spotty and ultimately that element shouldn't have been an actual `<button>` anyway if it did nothing without JS.

I liked it too but it seemed to work intermittently 😞